staging/src/k8s.io/apiserver/pkg/admission/interfaces.go

```diff
@@ -84,15 +98,15 @@ type MutationInterface interface {
 	Interface
 
 	// Admit makes an admission decision based on the request attributes
-	Admit(a Attributes) (err error)
+	Admit(a Attributes, o ObjectInterfaces) (err error)
 }
 
 // ValidationInterface is an abstract, pluggable interface for Admission Control decisions.
 type ValidationInterface interface {
 	Interface
 
 	// Validate makes an admission decision based on the request attributes.  It is NOT allowed to mutate
-	Validate(a Attributes) (err error)
+	Validate(a Attributes, o ObjectInterfaces) (err error)
```

Comment:
This silently disabled validation plugins of 3rdparty code bases. We should start caring about compatibility. We could have added `ObjectInterfaces` to the attributes without damaging third parties.

Reply:
Changing either interface would have broken compatibility. I'd strongly recommend type assertions downstream to ensure interface matching.